client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -2,18 +2,99 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import AutoDirection from 'components/auto-direction';
+import { getCurrentUser } from 'state/current-user/selectors';
 
 export class CommentDetailReply extends Component {
+	state = {
+		commentText: '',
+		haveFocus: false,
+	};
+
+	handleBlur = () => this.setState( { haveFocus: false } );
+
+	handleFocus = () => this.setState( { haveFocus: true } );
+
+	handleKeyDown = event => {
+		// Use Ctrl+Enter to submit comment
+		if ( event.keyCode === 13 && ( event.ctrlKey || event.metaKey ) ) {
+			event.preventDefault();
+			this.submit();
+		}
+	}
+
+	handleSubmit = event => {
+		event.preventDefault();
+		this.submit();
+	}
+
+	handleTextChange = event => this.setState( { commentText: event.target.value } );
+
+	submit = () => {
+		const comment = {
+			authorAvatarUrl: this.props.currentUser.avatar_URL,
```

Comment:
Unless we need the full object, maybe consider updating `mapStateToProps` to pass through the individual author props.